po/pl_PL.po

```diff
@@ -2,30 +2,33 @@ msgid ""
 msgstr ""
 "Project-Id-Version: moneymanagerex\n"
 "Report-Msgid-Bugs-To: developer@moneymanagerex.org\n"
-"POT-Creation-Date: 2019-02-03 17:46+0100\n"
-"PO-Revision-Date: 2019-02-03 17:46+0100\n"
-"Last-Translator: Tomasz Słodkowicz\n"
+"POT-Creation-Date: 2019-02-03 12:43+0300\n"
+"PO-Revision-Date: 2019-02-03 21:13\n"
+"Last-Translator: vomikan <nikolay.akimov@gmail.com>\n"
 "Language-Team: Polish\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "X-Generator: crowdin.com\n"
-"X-Poedit-SourceCharset: UTF-8\n"
+"X-Poedit-Basepath: ../src\n"
+"X-Poedit-KeywordsList: _;wxTRANSLATE\n"
+"X-Poedit-SearchPath-0: .\n"
+"X-Poedit-SearchPathExcluded-0: db/DB_Upgrade.h\n"
```

Comment:
no need to add this outdated Poedit settings

Reply:
I've updates mmex.pot file from sources by poedit tool. I don't know other and working for me way.

Reply:
Do you want a Windows version of update-po-files.sh? I think Poedit will be not enought to generate complete .pot file from all sources (see https://github.com/slodki/moneymanagerex/blob/29b05f50f658ab1e092c69a5fab9344f1a010fe0/util/update-po-files.sh#L22)

Reply:
Poedit is working excellent. Last update of pot has done by Poedit.
And it has interface with Crowdin. 
![image](https://user-images.githubusercontent.com/6836805/52196796-37d91d00-286e-11e9-8012-35d12fc62445.png)